api/sql/poiSQL/initialize.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS POIs CASCADE;
+
+CREATE TABLE POIs (
+    id SERIAL PRIMARY KEY,
+    name TEXT,
+    type ENUM
```

Comment:
I don't think ENUM is a valid postgres type, we'll have to define our own "POI type" enum on a line above (like between the drop and create or something).

See: [https://www.postgresql.org/docs/current/datatype-enum.html](https://www.postgresql.org/docs/current/datatype-enum.html)

Reply:
Good catch!! Thanks for including that documentation, I'll check it out!